docs/local_resource.md

```diff
@@ -57,3 +57,30 @@ For more on trigger mode, [see the docs](https://docs.tilt.dev/manual_update_con
 `auto_init=False` is currently only compatible with `TRIGGER_MODE_MANUAL`. If
 you'd like a local resource that runs automatically in response to file changes
 but does NOT run on `tilt up`, [let us know](https://tilt.dev/contact).
+
+## serve_cmd
```

Comment:
maybe worth it to elaborate on WHY we named it this -- because we imagine it being (at least in the platonic case) a command that runs a server. even if it's not always used for that, i think it'll help users to wrap their heads around the naming and what it's for if they have an example

Reply:
yeah, good point

though trying to justify it just makes it sound like a bad name :(